### ISSUE TITLE
No Translations for Global Settings?

### ISSUE THREAD

**Issue by LBYPatrick** [ziahamza/webui-aria2#332]
I've just tested this... No matter what language you've picked, descriptions for options in Global Settings would be always English. I checked the source code and...no one was ever working on this...

**Comment by ziahamza** [comment]
Yes actually its not there @LBYPatrick . But it can be added, but a lot of work. Will you be interested to write a translation for that? If you are interested then I can add the support infrastructure to help out :)

**Comment by LBYPatrick** [comment]
True It would be a lot of work...setting up a support infrastructure is beneficial for the future. I might be able to help out with Simplified Chinese after May 27th (My semester will end on that day).

**Comment by LBYPatrick** [comment]
@ziahamza I am ready for working on the translation work. Please start working on the framework.
